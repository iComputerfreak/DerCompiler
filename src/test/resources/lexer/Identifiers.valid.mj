foo_
_
____
a_b_c_d_e
adsf
as!df
11foo
