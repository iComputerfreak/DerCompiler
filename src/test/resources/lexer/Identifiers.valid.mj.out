identifier foo_
identifier _
identifier ____
identifier a_b_c_d_e
identifier adsf
identifier as
!
identifier df
integer literal 11
identifier foo
EOF
